src/App.tsx

```diff
@@ -1,16 +1,25 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [moviesServer, setMoviesServer] = useState<Movie[]>(moviesFromServer);
+  const onAdd = (newmovie:Movie) => {
+    setMoviesServer([...moviesServer, newmovie]);
```

Comment:
```suggestion
    setMoviesServer((moviesServer) => [...moviesServer, newmovie]);
```
Consider using a callback here to avoid bugs